<?xml version="1.0"?>



<launch>
  <group ns="small_scout_1">

    <node name="inference" pkg="src2_object_detection" type="inference.py" output="screen">
      <param name="print_to_terminal" type="bool" value="True"/>
    </node>

    <node name="plot" pkg="src2_object_detection" type="plot_boxes_real_time.py" output="screen">
    </node>
<!--
    <node name="object_estimation_service" pkg="src2_object_detection" type="object_estimation_service.py" output="screen">
      <param name="print_to_terminal" type="bool" value="False"/>
    </node>

    <node name="approach_base_station_service" pkg="src2_object_detection" type="approach_base_station_service.py" output="screen">
      <param name="print_to_terminal" type="bool" value="True"/>
    </node>

    <node name="align_base_station_service" pkg="src2_object_detection" type="align_base_station_service.py" output="screen">
      <param name="print_to_terminal" type="bool" value="True"/>
    </node> -->

  </group>
</launch>
